packit_service/worker/whitelist.py

```diff
@@ -195,23 +195,24 @@ def check_and_report(self, event: Optional[Any], project: GitProject) -> bool:
         :param project: GitProject
         :return:
         """
-        account_name = None
+
         # TODO: modify event hierarchy so we can use some abstract classes instead
-        if isinstance(
-            event, (PullRequestEvent, PullRequestCommentEvent, IssueCommentEvent)
-        ):
-            account_name = event.github_login
-            assert account_name, f"Failed to get account_name from {type(event)}"
         if isinstance(event, ReleaseEvent):
             account_name = event.repo_namespace
             assert account_name, f"Failed to get account_name from {type(event)}"
+            if not self.is_approved(account_name):
+                logger.info(f"Refusing release event on not whitelisted repo namespace")
+                return False
+            return True
         if isinstance(
             event,
             (CoprBuildEvent, TestingFarmResultsEvent, DistGitEvent, InstallationEvent),
         ):
             return True
+        if isinstance(event, (PullRequestEvent, PullRequestCommentEvent)):
+            account_name = event.github_login
+            assert account_name, f"Failed to get account_name from {type(event)}"
```

Comment:
`assert` should not be used in the production code -- it was meant for testing/debugging and python code that is run with some optimisations (e.g. `-O` option) can skip the `assert` without executing. 

Can you please fix that also in other occurrences?